pages/scheduler/basic-usage.md

```diff
@@ -74,40 +71,35 @@ import {
    }
 ```
 
-* After authenticating the visualize.js object ("V object") is returned. This is used to
-  interact with the internal API of visualize.js. Store a reference to this object in a global scope to perform other
-  operations with visualize.js such as loading a report viewer.
+* After authenticating, the `VisualizeClient` object (or simply `v` object) is returned. It is used to
+  interact with the Visualize.js API. Store a reference to this object in a global scope of your application to perform other operations with Visualize.js, such as loading a report.
 
-More info about the JVTools can be found in this [guide]({{site.baseurl}}/pages/tools/loading-vizjs).
+More details about Visualize.js loader can be found at [loading visualize.js]({{site.baseurl}}/pages/tools/loading-vizjs).
 
-## Rendering the scheduler plugin
+## Render the Scheduler
 
 ### The JavaScript approach
 
 #### Import
 ```js 
 import { renderScheduler, SchedulerConfig } from "@jaspersoft/jv-scheduler";
 ```
-Scheduler component has a method `renderScheduler` that accepts 3 parameters: container, visualize object, scheduler configuration.
-`SchedulerConfig` is a typing for scheduler configuration object.
 
-```js
-renderScheduler(
-    document.getElementById("my-container"),
-    visualize_object, 
-    schedulerUIConfig,
-);
-
-```
-
-* `container`: (DivElement) <div> element from the DOM where the scheduler should be rendered
-* `visualize_object`: (VisualizeClient) visualization object
-* `schedulerUIConfig`: (SchedulerConfig) object containing configuration for scheduler look and feel and event handling. [reference]({{site.baseurl}}/pages/scheduler/configuration.html#configuration)
+Scheduler package provides a
+`renderScheduler(v: VisualizeClient, container: HTMLElement, config: SchedulerConfigProps): void` method.
```

Comment:
This method is missing the '`uri:string`' parameter.
It has been added recently to match the same interface as in `InputControls` package.

Reply:
Done